### ISSUE TITLE
feature: Heading styling enhancement

### ISSUE THREAD

**Issue by Krystian-KF** [MeanderingProgrammer/render-markdown.nvim#121]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

It would be amazing to have the same stylings for headings as for a code block.  The ones missing are:
- left pad
- right pad
- min_width

![image](https://github.com/user-attachments/assets/9c0f56f0-a368-4541-8414-7900bcb73d0b)

That would give ultimate flexibility how to style our headings, making it possible to line them up with code blocks and dash lines for visual consistency. I believe that it would even  enable us to centred the heading as well.

![image](https://github.com/user-attachments/assets/a320ed31-1860-4324-b957-50234dddade5)

### Describe alternatives you've considered

Just having min_width would go a long way. As an alternative I am using a blank character (Braille Pattern Blank: U+2800) to exdend the heading.ㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤㅤ

**Comment by MeanderingProgrammer** [comment]
Feature added here: https://github.com/MeanderingProgrammer/render-markdown.nvim/commit/6392a5dfa10f367e10fe58ea9c2faf3179b145d5

Works in the same way as code blocks, give it a shot!

**Comment by Krystian-KF** [comment]
Amazing, works as a charm!

Thank you very much for quick response and implementation.

I cannot say enough how thankful I am to you for this plugin! I have tried some few other options before but this is the only one that gives me almost limitless level of customization and enables me to have my notes look as close as possible (being in a Terminal) to how I configured and display them in Obsidian.

Thank You.
